infra/common/src/test/java/org/apache/shardingsphere/infra/database/metadata/url/StandardJdbcUrlParserTest.java

```diff
@@ -81,4 +81,23 @@ public void assertParseMicrosoftSQLServerJdbcUrl() {
     public void assertParseIncorrectURL() {
         new StandardJdbcUrlParser().parse("jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DATABASE_TO_UPPER=false;MODE=MySQL");
     }
+
+    @Test
+    public void assertParseTestContainersJDBCUrl() {
+        assertThat(new StandardJdbcUrlParser().parse("jdbc:tc:mysql:5.7.34:///demo_ds").getDatabase(), is("demo_ds"));
+        assertThat(new StandardJdbcUrlParser().parse("jdbc:tc:postgresql:9.6.8:///demo_ds").getDatabase(), is("demo_ds"));
+        assertThat(new StandardJdbcUrlParser().parse("jdbc:tc:postgis:9.6-2.5:///demo_ds").getDatabase(), is("demo_ds"));
+        assertThat(new StandardJdbcUrlParser().parse("jdbc:tc:timescaledb:2.1.0-pg13:///demo_ds").getDatabase(), is("demo_ds"));
+        assertThat(new StandardJdbcUrlParser().parse("jdbc:tc:trino:352://localhost/memory/default").getDatabase(), is("memory/default"));
```

Comment:
I've never used trino , I'm assuming that's the correct database name.

Reply:
That's ok. It's in the [test container examples]( https://www.testcontainers.org/modules/databases/jdbc/#jdbc-url-examples ).
But I didn't find all of them. Could you add test container example links in PR page?

Reply:
All the examples are from https://www.testcontainers.org/modules/databases/jdbc/#jdbc-url-examples . Is there anything missing?

Reply:
OK, I just had a glance at the page and missed some of them